pandas/core/groupby/grouper.py

```diff
@@ -71,52 +71,18 @@ class Grouper:
         Currently unused, reserved for future use.
     **kwargs
         Dictionary of the keyword arguments to pass to Grouper.
-    key : str, defaults to None
```

Comment:
These should remain

Reply:
I have added these back with an additional indentation else

`scripts/validate_docstrings.py --format=actions pandas.Grouper` is giving the following error.

```
################################################################################
################################## Validation ##################################
################################################################################

1 Errors found for `pandas.Grouper`:
        PR02    Unknown parameters {'key', 'label', 'sort', 'closed', 'convention', 'dropna', 'freq', 'level', 'offset', 'origin'}

```

With indentation its working fine

```
################################################################################
################################## Validation ##################################
################################################################################

Docstring for "pandas.Grouper" correct. :)
```

Reply:
let me know if this is fine now.

Reply:
I don't think this should be adjusted just to appease the current docstring checker. IMO it looks a bit confusing that this is nested under kwargs.

I would suggest avoid touching these parameters as it probably would be better to make these keywords permanement over `*args, **kwargs`

Reply:
restored it to upstream main.